x-pack/plugins/triggers_actions_ui/public/application/components/health_check.tsx

```diff
@@ -94,15 +94,11 @@ export const HealthCheck: React.FunctionComponent<Props> = ({
 };
 
 interface PromptErrorProps {
-  docLinks: Pick<DocLinksStart, 'ELASTIC_WEBSITE_URL' | 'DOC_LINK_VERSION'>;
+  docLinks: Pick<DocLinksStart, 'links'>;
```

Comment:
**optional nit:** I'm not sure why it did a `Pick<...>` before 🤔 looks like we might be ok removing it.

```suggestion
  docLinks: DocLinksStart;
```